pkg/security/security_profile/dump/load_controller.go

```diff
@@ -58,7 +59,7 @@ func NewActivityDumpLoadController(adm *ActivityDumpManager) (*ActivityDumpLoadC
 	}, nil
 }
 
-func (lc *ActivityDumpLoadController) getDefaultLoadConfig() *model.ActivityDumpLoadConfig {
+func (lc *ActivityDumpLoadController) getDefaultLoadConfigs() (map[containerutils.CGroupManager]*model.ActivityDumpLoadConfig, error) {
```

Comment:
As this func is called multiple times in case of silent workloads, could we initialize only once the default configs?

Reply:
Done